x/programs/cmd/simulator/cmd/config.go

```diff
@@ -33,6 +28,8 @@ const (
 	/// function call. A program's function can internally optionally call other
 	/// functions including program to program.
 	EndpointExecute Endpoint = "execute"
+	/// Create a new program on-chain
+	EndpointCreateProgram Endpoint = "createprogram"
```

Comment:
```suggestion
	EndpointCreateProgram Endpoint = "create_program"
```

Reply:
don't think we should use snake_case here since its not `go` convention + enum types are UpperCamelCase anyways